sandbox/public/perf/personalization.html

```diff
@@ -34,7 +34,9 @@
       const onSendEvent = () => {
         alloy("sendEvent", {
           renderDecisions: true,
```

Comment:
I would create a separate page and leave this one as is.

Reply:
@ninaceban I've made changes to the sandbox and the tests to prioritize the new `personalization` config, as in the meeting we had on this change it was decided that going forward it is now the recommended way to set `decisionScopes`, and the top-level `decisionScopes` config is now a deprecated one. So, anyone looking through the code/test/sandbox should be aware of this.
@jfkhoury please let me know if my understanding is wrong and we should revert all the mods relating to this.

Reply:
In the current PR we should only add the `personalization` config to support surfaces, in a following PR we will address the deprecation strategy. Even though we are recommending that the new way of querying personalized content is by sending the `decisionScopes` in the `personalization` config. We still need to be able to validate that the existing functionality works properly. That is why it is so important that we leave the sandbox app and functional tests as are, and add new page/functional tests to test the new behavior.
cc: @jfkhoury

Reply:
This comment addresses all the `Sandbox` related comments:

We need to tackled to goals here:

1. Test the new `personalization` option.
2. Confirm that the `decisionsScopes` option still works, until we decide to deprecate it. (By the way we need a ticket to do so in v 3.0)

So I suggest we do the following:

1. I agree with @ninaceban, let's leave these pages alone for now; maybe add an inline comment that this `decisionScopes` option will be deprecated in v 3.0.
2. Let's add a brand new AJO Surfaces/Web Channel Sandbox page that showcases the surfaces support and use the new `personalization` option.
3. I would like to see a test/sandbox page/something that showcases the usage of `decisionsScopes` and `personalization.decisionsScopes` and see what happens.

Thank you @XDex !!

Reply:
@jfkhoury It's probably a misunderstanding on my end of the decisions taken in the meeting we had on nesting `decisionScopes` in `personalization` - I thought we'll be deprecating the top-level `decisionScopes` in my PR and not a separate one. Let me revert the sandbox mods and add inline comments to the API as you suggest.
Regarding p.2 - I've already added two e2e functional tests for AJO and a dedicated page to the sandbox, please take a look.
On p.3 - I've also previously added a test where we're checking that the top-level `decisionScopes` option is working as expected, either standalone or together with the new `personalization.decisionScopes`